assets/js/graphql/events.js

```diff
@@ -0,0 +1,21 @@
+import gql from 'graphql-tag';
+
+export const EventFragment = gql`
```

Comment:
I usually think of camel-cased variables in javascript to mean classes. should this be a screaming snake case like `EVENTS_SUBSCRIPTION`?